packages/@okta/vuepress-site/docs/guides/authenticators-custom-authenticator/main/ios/sdksteps.md

```diff
@@ -0,0 +1,316 @@
+Use the Devices SDK to enable your app to verify the identity of a user by responding to notifications from a custom authenticator. To setup and configure your app:
+- Add the SDK to your app
+- Configure the required capabilities
+- Register for notifications
+- Initialize the SDK client
+
+Next, register, or enroll the device to receive identity verification notifications for the custom authenticator. Your app needs to support user sign in and request the correct permissions, or scopes. Once the device is enrolled the app can respond to user verification notifications (challenges).
+
+If needed, you can also unenroll the device, either locally, or both locally and on the server.
+
+> **Note:** The iOS code snippets assume that there is a singleton class called `DeviceSDKManager` that manages interactions with the Devices SDK. The singleton contains any required state information, such as the current SDK client, the APNs token for the current launch of the app, utility functions, and more. Your app may use a different way of interacting with the parts of the Devices SDK.
+
+The following image shows how data flows through the Devices SDK:
+
+<div class="full">
+
+![Custom authenticator flowchart](/img/authenticators/authenticators-custom-authenticator-data-flow.png)
+
+</div>
+
+### Add sign-in to your app
+
+Add sign in using the [Okta mobile Swift SDK](https://github.com/okta/okta-mobile-swift)
+or if you're already using it, the [Okta IDX SDK](https://github.com/okta/okta-idx-swift). For more information on signing users in to your app, see [Sign users in to your mobile app using the redirect model](https://developer.okta.com/docs/guides/sign-into-mobile-app-redirect/ios/main/).
+
+Add the following strings to the space-delimited list of scopes in the `Okta.plist` file after you've configured your app to enable sign-in. These scopes request the additional permissions for the access token that are required by the Devices SDK:
+- `okta.authenticators.manage.self`
```

Comment:
The only answer I can give is that this is what engineering requested and what the Sample app does. If there are different use cases I suspect the answer is still to add all 3. Otherwise it requires signing out, reinitializing the sign-in client, and signing in again to acquire a new token. I'll leave the modified things as is.